pkg/local_object_storage/metabase/db.go

```diff
@@ -290,6 +292,14 @@ func (db *DB) SetLogger(l *zap.Logger) {
 	db.log = l
 }
 
+// SaveLastResyncEpoch saves the current epoch as the last one
+// when metabase was re-synchronized, to DB and to Info.
+func (db *DB) SaveLastResyncEpoch() {
+	currEpoch := db.epochState.CurrentEpoch()
+	db.lastResyncEpoch = currEpoch
+	db.info.LastResyncEpoch = currEpoch
```

Comment:
how can we use it? every time we restart application this field is lost. i planned to extend metabase storage with a number that is filled every reset and is read every start